.github/workflows/integration-test.yaml

```diff
@@ -0,0 +1,286 @@
+       go test -v -timeout 1800s -run CephMgrSuite github.com/rook/rook/tests/integration
+
+  TestCephHelmSuite:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+
+    - name: setup golang
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15
+
+    - name: setup minikube
+      uses: manusa/actions-setup-minikube@v2.0.1
+      with:
+        minikube version: 'v1.13.1'
+        kubernetes version: 'v1.19.2'
+        driver: docker
+        start args: --memory 6g --cpus=2
+
+    - name: check k8s cluster status
+      run: |
+        kubectl cluster-info
+        kubectl get pods -n kube-system
+        for i in $(docker ps|awk '/minikube/ {print $1}'); do docker cp /sbin/lvm "$i":/sbin/lvm; done
+
+    - name: use local disk
+      run: |
+        sudo swapoff --all --verbose
+        sudo umount /mnt
+        # search for the device since it keeps changing between sda and sdb
+        sudo wipefs --all --force /dev/$(lsblk|awk '/14G/ {print $1}'| head -1)1
+        sudo lsblk
+
+    - name: build rook
+      run: |
+       # set VERSION to a dummy value since Jenkins normally sets it for us. Do this to make Helm happy and not fail with "Error: Invalid Semantic Version"
+       GOPATH=$(go env GOPATH) make clean && make -j$nproc IMAGES='ceph' VERSION=0 build
+       docker images
+       docker tag $(docker images|awk '/build-/ {print $1}') rook/ceph:master
+
+    - name: TestCephHelmSuite
+      run: |
+       tests/scripts/minikube.sh helm
+       tests/scripts/helm.sh up
+       sed -i "s|deviceFilter:  ''|deviceFilter: $(lsblk|awk '/14G/ {print $1}'| head -1)|g" tests/framework/installer/ceph_manifests.go
+       # sed -i 's|snapshotterVersion = "v2.1.0"|snapshotterVersion = "v3.0.0"|g' tests/framework/utils/snapshot.go
+       go test -v -timeout 1800s -run CephHelmSuite github.com/rook/rook/tests/integration
+
+    - name: The job has failed
+      if: ${{ failure() }}
+      run: | 
+       cd /home/runner/work/rook/rook/tests/integration/_output/tests/
+       ls
+       FILE=$(find . ! -name '*_previous_*' -name 'TestCephHelmSuite_localhost_helm-ns_rook-ceph-operator-*.log')
+       test -f $FILE && cat *       
+      
+  TestCephMultiClusterDeploySuite:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+
+    - name: setup golang
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15
+
+    - name: setup minikube
+      uses: manusa/actions-setup-minikube@v2.0.1
+      with:
+        minikube version: 'v1.13.1'
+        kubernetes version: 'v1.19.2'
+        driver: docker
+        start args: --memory 6g --cpus=2
+
+    - name: check k8s cluster status
+      run: |
+        kubectl cluster-info
+        kubectl get pods -n kube-system
+        minikube ssh -- sudo mkdir -p /var/lib/rook/rook-integration-test/mon1 /var/lib/rook/rook-integration-test/mon2 /var/lib/rook/rook-integration-test/mon3
```

Comment:
You don't need that if you use the `bare` provider. Why do you use `docker` all of the sudden?

Reply:
Yeah, I'm using `docker` only for `TestCephMultiClusterDeploySuite ` this test was failing earlier with some reason...I don't remember the error but while running the script `localPathPV.sh`. I'll try again with `bare` and see if it's working.

Reply:
You might as well run the `localPathPV.sh` before it helps. bare should work as all the other canary scenario use it successfully.